Port Install on Mac OSX

I am trying to install vg on a Mac with  El Capitan OS X.

The first issue is the port install of the redland-utils which the README indicates should be:
`sudo port install  redland-utils`
but this generates:
Error: Port redland-utils not found

However, 
`sudo port install redland `
does install. So it looks like the vg README needs to be updated.

The other issue is related to bison. The port install places the most up to date version of bison in /opt/local/bin. This directory is ahead of /usr/bin in the PATH where the old 2.3 bison is installed. However, this error is generated when compiling.  To get around this, I could update or replace the old bison in /usr/bin but was wondering if there was a cleaner way to achieve this in the source_me.sh  or make scripts.

`. ./source_me.sh && make`
cd deps/raptor/build && cmake .. && /Applications/Xcode.app/Contents/Developer/usr/bin/make && cp src/libraptor2.a /Users/farrell/github/vg/lib && mkdir -p /Users/farrell/github/vg/include/raptor2 && cp src/*.h /Users/farrell/github/vg/include/raptor2
CMake Error at /opt/local/share/cmake-3.5/Modules/FindPackageHandleStandardArgs.cmake:148 (message):
  Could NOT find BISON: Found unsuitable version "2.3", but required is at
  least "3" (found /usr/bin/bison)
Call Stack (most recent call first):
  /opt/local/share/cmake-3.5/Modules/FindPackageHandleStandardArgs.cmake:386 (_FPHSA_FAILURE_MESSAGE)
  /opt/local/share/cmake-3.5/Modules/FindBISON.cmake:226 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  CMakeLists.txt:33 (FIND_PACKAGE)

--- a/README.md
+++ b/README.md
@@ -40,7 +40,7 @@ Then build with `. ./source_me.sh && make static`, and run with `./bin/vg`.
 
 VG won't build with XCode's compiler (clang), but it should work with GCC 4.9.  One way to install the latter (and other dependencies) is to install [Mac Ports](https://www.macports.org/install.php), then run:
 
-    sudo port install gcc49 libtool jansson jq cmake pkgconfig autoconf automake libtool coreutils samtools redland-utils
+    sudo port install gcc49 libtool jansson jq cmake pkgconfig autoconf automake libtool coreutils samtools redland bison
 
 To make GCC 4.9 the default compiler, run (use `none` instead of `mp-gcc49` to revert back):
 

